src/slider.test.events.multiple.ts

```diff
@@ -0,0 +1,896 @@
+      cancelable: true,
+    }),
+  );
+
+  await host.updateComplete;
+
+  expect(spy.callCount).to.equal(1);
+});
+
+it('dispatches a "change" event dragging and letting go of the maximum handle', async () => {
+  const host = await fixture<Slider>(
+    html`<glide-core-slider label="Label" multiple></glide-core-slider>`,
+  );
+
+  const spy = sinon.spy();
+  host.addEventListener('change', spy);
+
+  const sliderTrack = host.shadowRoot?.querySelector('[data-test="slider"]');
+
+  const maximumHandle = host.shadowRoot?.querySelector(
+    '[data-test="maximum-handle"]',
+  );
+
+  const trackRect = sliderTrack?.getBoundingClientRect();
+
+  assert(trackRect);
+  assert(maximumHandle);
+
+  maximumHandle.dispatchEvent(
+    new MouseEvent('mousedown', {
+      bubbles: true,
+      cancelable: true,
+    }),
+  );
+
+  await aTimeout(0);
+
+  document.dispatchEvent(
+    new MouseEvent('mousemove', {
+      bubbles: true,
+      cancelable: true,
+      clientX: trackRect.left + trackRect.width * 0.7,
+    }),
+  );
+
+  await aTimeout(0);
+
+  expect(spy.callCount).to.equal(0);
+
+  document.dispatchEvent(
+    new MouseEvent('mouseup', {
+      bubbles: true,
+      cancelable: true,
+    }),
+  );
+
+  await host.updateComplete;
+
+  expect(spy.callCount).to.equal(1);
+});
+
+it('dispatches an "input" and "change" event clicking on the minimum side of the track', async () => {
+  const host = await fixture<Slider>(
+    html`<glide-core-slider label="Label" multiple></glide-core-slider>`,
+  );
+
+  const inputSpy = sinon.spy();
+  const changeSpy = sinon.spy();
+
+  host.addEventListener('input', inputSpy);
+  host.addEventListener('change', changeSpy);
+
+  const sliderTrack = host.shadowRoot?.querySelector('[data-test="slider"]');
+  const trackRect = sliderTrack?.getBoundingClientRect();
+
+  assert(trackRect);
+  assert(sliderTrack);
+
+  // Click on the track at the 10% position, which is closer
+  // to the minimum handle than the maximum handle.
+  sliderTrack.dispatchEvent(
+    new MouseEvent('click', {
+      bubbles: true,
+      cancelable: true,
+      clientX: trackRect.left + trackRect.width * 0.1,
+    }),
+  );
+
+  await host.updateComplete;
+
+  expect(inputSpy.callCount).to.equal(1);
+  expect(changeSpy.callCount).to.equal(1);
+});
+
+it('dispatches an "input" and "change" event clicking on the maximum side of the track', async () => {
+  const host = await fixture<Slider>(
+    html`<glide-core-slider label="Label" multiple></glide-core-slider>`,
+  );
+
+  const inputSpy = sinon.spy();
+  const changeSpy = sinon.spy();
+
+  host.addEventListener('input', inputSpy);
+  host.addEventListener('change', changeSpy);
+
+  const sliderTrack = host.shadowRoot?.querySelector('[data-test="slider"]');
+  const trackRect = sliderTrack?.getBoundingClientRect();
+
+  assert(trackRect);
+  assert(sliderTrack);
+
+  // Click on the track at the 90% position, which is closer
+  // to the maximum handle than the minimum handle.
```

Comment:
The "click" part is explicit in the code:

```suggestion
  // The 90% position is closer to the maximum handle than the minimum handle.
```